dashboard/v1.1/src/pages/Config/Config.tsx

```diff
@@ -143,77 +149,145 @@ const Config: FC<{}> = () => {
-                      ) : (
-                        <Tooltip title="Edit" style={{ cursor: 'pointer' }}>
-                          <EditIcon onClick={() => handleToggle(test)} />
-                        </Tooltip>
-                      )}
-                    </Grid>
-                  </Grid>
-                  <IntervalDetails
-                    reFetch={fetchConfigIntervals}
-                    toggleComponentView={(name: string) => handleToggle(name)}
-                    toggleResults={toggleResults}
-                    durationValue={duration}
-                    intervalName={test}
-                  />
-                  <Typography variant="body1">{unit}</Typography>
-                </CardContent>
-              </Card>
-            </Grid>
-          );
-        })}
-      </Grid>
-      <SearchTable
-        title=""
-        columns={columns}
-        data={getTableRoutes(configRoutes)}
-        editable={{
-          onRowUpdate: (newData, oldData) =>
-            new Promise((resolve, reject) => {
-              setTimeout(() => {
-                // const index = oldData.tableData.id;
-
-                resolve();
-              }, 1000);
-            }),
-          onRowDelete: oldData =>
-            new Promise((resolve, reject) => {
-              setTimeout(() => {
-                // const index = oldData.tableData.id;
-
-                resolve();
-              }, 1000);
-            })
-        }}
+                    <IntervalDetails
+                      reFetch={fetchConfigIntervals}
+                      toggleComponentView={(name: string) => handleToggle(name)}
+                      toggleResults={toggleResults}
+                      durationValue={duration}
+                      intervalName={test}
+                    />
+                    <Typography variant="body1" align="center">
+                      {unit}
+                    </Typography>
+                  </CardContent>
+                </Paper>
+              </Grid>
+            );
+          })}
+        </Grid>
+      </Paper>
+      <EditModal
+        isOpen={editModalOpen}
+        setOpen={(open: boolean) => setEditModalOpen(open)}
+        selectedRoute={selectedRow}
+        updateConfigRoutes={route => updateConfigRoutes(route)}
       />
+      <Suspense fallback={<CircularProgress disableShrink />}>
+        <Paper elevation={0}>
+          <SearchTable
+            title="URL endpoints"
+            columns={columns}
+            data={getTableData(Array.from(configRoutes))}
+            actions={[
+              {
+                icon: tableIcons.Edit,
+                tooltip: 'Edit Route',
+                onClick: (event, rowData: TableRowData) => {
+                  setSelectedRow({
+                    route: rowData.route,
+                    options: configRoutes.get(rowData.route)
+                  });
+                  setEditModalOpen(!editModalOpen);
+                }
+              }
+            ]}
```

Comment:
please see whether this can be declared in `handles.ts` and then imported here.

Reply:
As an additional prop here, I'd suggest this to be here as it is